docs/usage/8-authentication.md

```diff
@@ -145,11 +145,13 @@ Finally, we need to pass our middleware to the Starlite constructor:
 
 ```python title="my_app/main.py"
 from starlite import Starlite
+from starlite.middleware.base import DefineMiddleware
 
 from my_app.security.authentication_middleware import JWTAuthenticationMiddleware
 
+auth_mw = DefineMiddleware(JWTAuthenticationMiddleware, excluded="schema")
```

Comment:
Is this the correct way to define middleware that take arguments?

Reply:
Yup. 

Although there is a JWTAuth.middkeware that dies it for you